bundles/org.openhab.core.io.rest.auth/bnd.bnd

```diff
@@ -1,2 +1 @@
 Bundle-SymbolicName: ${project.artifactId}
```

Comment:
I think the whole file can now be removed since its purpose was to add the activator.